docs/upgrade/automated_upgrade.md

```diff
@@ -54,7 +54,6 @@ spec:
        # When using k8s version 1.19 or older, swap control-plane with master
        - {key: node-role.kubernetes.io/control-plane, operator: In, values: ["true"]}
   serviceAccountName: system-upgrade
-  cordon: true
```

Comment:
I'd prefer to remove the one from up above, for consistency with the agent Plan spec the follows.